recipe/meta.yaml

```diff
@@ -1,33 +1,33 @@
 {% set name = "CairoSVG" %}
-{% set version = "2.4.2" %}
+{% set version = "1.0.22" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 4e668f96653326780036ebb0a9ff2bb59a8443d7bcfc51a14aab77b57a8e67ad
+  sha256: f66e0f3a2711d2e36952bb370fcd45837bfedce2f7882935c46c45c018a21557
 
 build:
   number: 0
   noarch: python
   entry_points:
-    - cairosvg = cairosvg.__main__:main
+    - cairosvg = cairosvg.__init__:main
   script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   host:
-    - python >=3.5
+    - python <3
```

Comment:
is this constraint really required? or could it be simply `python`? (same below in `run` section)

Reply:
Hmm, [setup.py at the 1.0.22 tag](https://github.com/Kozea/CairoSVG/blob/1.0.22/setup.py) only declares support up to Python 3.4.  Could be risky to try it on `python>3.4`, and the whole point of bothering to take this back to 1.0.22 was to make sure it was to offer a solution for `python<3`.  I see now that this might also be a solution for `python<=3.4`, since the latest versions are pinned to `python>=3.5`.  I'll try changing this to python<=3.4 and see how it goes, but  I want to make sure the CI services also test on python<3

Reply:
I think I get it now. cairosvg is pure python, but its dependencies are not. I should let the package manager figure the dependencies out: We just need to tell conda-forge to build and keep 1.0.22. That way, the dependency solver will find it for python<3. If there are problems in the dependencies not being available for some OSes for python<2, the dependency solver will notice. Issues will be reported this or the dependency feedstocks and that can be fixed in the feedstocks of the respective dependencies.

This still allows a user to try and use cairosvg=1.0.22 in a python 3 environment if they see a need to, since it's not explicitly forbidden by cairosvg.

That still leaves the question of the optional dependencies. I think we should include them when a user asks conda to install cairosvg: It is most likely to work when all of the opitonal deps are there. On top of them, these optional deps turned into mandatory ones in later releases, so the maintainers of cairosvg thought the features were important enough to include. It also avoids the problem of the user needing to figure out that, even though on python 3 they just have to conda install cairosvg, on python 2 they need to conda install cairosvg lxml cssselect tinycss because their code uses some feature of cairosvg that required them. What do you think? I'll force push a clean commit history to PR#23 and reopen that PR so these experiments remain available. I can't reopen a pull request after force pushing, so I'll pop it here and open it for review.